torch/quantization/fuse_modules.py

```diff
@@ -52,44 +61,68 @@ def fuse_conv_bn_relu(conv, bn, relu):
         return torch_fused.ConvReLU2d(
             torch.nn.utils.fusion.fuse_conv_bn_eval(conv, bn), relu)
 
+def _get_module(model, submodule_key):
+    submodule = ''
+    tokens = submodule_key.split('.')
+    cur_mod = model
+    for s in tokens:
+        if s.isdigit():
```

Comment:
I just tried and I don't think you need this - getattr seems to work just fine on string keys that are numbers